examples/vue-shopify-account/README.md

```diff
@@ -0,0 +1,142 @@
+# Accounts example
+
+Utilizing Shopify Storefront API and Multipass
+
+---
+
+## Summary
+
+We want a simple solution to handling customer accounts that doesn't involve maintaining code in Shopify's Theme. This example should give us all of the basic account functionality that we need.
+
+If you want to have a better understanding of the queries and mutations that we are utlizing, check out the [Shopify API documentation](https://shopify.dev/api).
+
+## Prerequisites
+
+- Must be on the [Shopify Plus](https://www.shopify.com/plus/enterprise-ecommerce) plan and have [Multipass](https://help.shopify.com/en/api/reference/plus/multipass) enabled.
+- You'll need your store's:
+  - Storefront API token
+  - Multipass secret
+
+## Setup
+
+1. add a few items to our `.env` file:
+
+```sh
+VITE_MYSHOPIFY_DOMAIN=""
+VITE_SHOPIFY_TOKEN=""
+VITE_SHOPIFY_ADMIN_PASSWORD=""
+VITE_SHOPIFY_MULTIPASS_SECRET=""
+```
+
+this specific project uses vite for frontend tooling, in your own project if you aren't using vite, the `VITE_` prefix is unnecessary.
+
+2. `npm install`
+
+3. you will also need the [vercel CLI](https://vercel.com/cli) installed in order to test serverless functions
+
+## Serverless Functions
+
+To use Multipass and the address form for account pages this project relies on two packages:
+
+- [multipassify](https://github.com/beaucoo/multipassify)
+- [countrycitystatejson](https://github.com/khkwan0/countryCityStateJson)
+
+Both of these packages are _large_ and can add a lot to your client bundle, so using them only in serverless functions keeps the client a little more lightweight. But if you want another reason, it also keeps your Multipass secret outside of client code.
+
+This project includes a folders for serverless functions:
+
+- Vercel in [api/\*](./api)
+
+When testing your serverless functions locally make sure to use the platform's CLI ([Vercel CLI](https://vercel.com/docs/cli). NPM scripts for running the project with these CLI's are provided in the `package.json`.
+
+## Functions
+
+All the functions to interact with the API live in the `/components/AccountProvider.vue` file and the graphql queries are in the `/gql/index.js`.
+
+We are using [the composition API](https://vuejs.org/guide/extras/composition-api-faq.html) for the state management but this can be integrated into any state management system.
```

Comment:
```suggestion
We are using [the composition API](https://vuejs.org/guide/extras/composition-api-faq.html) for state management but this can be integrated into any state management system.
```